website/assets/js/pages/handbook/basic-handbook.page.js

```diff
@@ -46,6 +46,24 @@ parasails.registerPage('basic-handbook', {
       }
     }
 
+    // If this is the handbook landing page, iterate through markdown pages in `this.markdownPages`
+    // if(this.isHandbookLandingPage){
+    //  for (page of markdownpages) {
+    //  if the page is a handbook page, create an object that contains the page's title, pageOrderInSection, and the array of markdown links on that page
+    //  example: {
+    //    pageTitle: 'People',
+    //    pageRank: 400,
```

Comment:
Consider just making it alphabetical here, no rank.  (Keep in mind this will all get more complex later if/when recursive, so maybe better to keep it simple now)

Also fwiw "page rank" is ["a thing"](https://en.wikipedia.org/wiki/PageRank) that has connotations, so picking a different phase could make sense